components/Hooks/index.tsx

```diff
@@ -102,3 +89,13 @@ export const useWindowWidth = () => {
   }, [width])
   return width
 }
+
+export const useInternationalization = (key: string): string => {
+  const { locale } = useRouter()
+  return webDictionary[locale][key]
+}
+
+export const useTranslation = (): WebTranslations => {
+  const { locale } = useRouter()
+  return webDictionary[locale]
+}
```

Comment:
`useTranslation` and `useInternationalization` seem like synonyms, but they are implemented differently. I think you should rename these to `getTranslations` and `getTranslationByKey`, respectively. And to clean things up, you could reference `getTranslations` within `getTranslationByKey`, as they share some logic.